tools/d2g/d2g.go

```diff
@@ -443,12 +404,18 @@ func runCommand(containerName string, dwPayload *dockerworker.DockerWorkerPayloa
 	// note, dwImage.String() is already shell escaped
 	command.WriteString(" " + dwImage.String())
 	command.WriteString(" " + shell.Escape(dwPayload.Command...))
-	return command.String(), nil
+	return [][]string{
+		{
+			"/usr/bin/env",
```

Comment:
is it still necessary to use env bash to run this? It looks like it is just `docker run` now, or can it contain more commands?

Reply:
i think this is just lifting what we had, essentially. there will be further cleanup in future PRs i believe that simplifies this and several other things (such as not shelling out to a script to perform logic that can be performed directly in go, for logic that is now part of the feature, and no longer part of the task commands).